third-party/realdds/scripts/devices.py

```diff
@@ -0,0 +1,52 @@
+# License: Apache 2.0. See LICENSE file in root directory.
+# Copyright(c) 2023 Intel Corporation. All Rights Reserved.
+
+from argparse import ArgumentParser
+args = ArgumentParser()
+args.add_argument( '--debug', action='store_true', help='enable debug mode' )
+args.add_argument( '--quiet', action='store_true', help='No output; just the minimum FPS as a number' )
+def time_arg(x):
+    t = int(x)
+    if t <= 0:
+        raise ValueError( f'--time should be a positive integer' )
+    return t
+args.add_argument( '--time', metavar='<seconds>', type=time_arg, default=2, help='seconds to gather devices for (default 2)' )
+def domain_arg(x):
+    t = int(x)
+    if t <= 0:
```

Comment:
Why not check `> 232`? Same for other script

Reply:
copy paste :)
Fixed.